Support for GNU/kFreeBSD, by Aurelien Jarno and others.

--- a/configure.ac
+++ b/configure.ac
@@ -1127,6 +1127,7 @@
 		#endif
 		#include <net/bpf.h>
 	    ])
+	    LIBS="$LIBS -lrt"
 	;;
 
 pf)
--- a/pcap-bpf.c
+++ b/pcap-bpf.c
@@ -26,6 +26,7 @@
 #include <sys/param.h>			/* optionally get BSD define */
 #include <sys/socket.h>
 #include <time.h>
+#include <sys/kern/time.h>
 /*
  * <net/bpf.h> defines ioctls, but doesn't include <sys/ioccom.h>.
  *
