Support for GNU/kFreeBSD, by Aurelien Jarno and others.

--- a/configure.ac
+++ b/configure.ac
@@ -108,6 +108,7 @@
 AC_CHECK_HEADERS(linux/types.h)
 AC_CHECK_HEADERS(linux/if_packet.h netpacket/packet.h netpacket/if_packet.h)
 AC_CHECK_HEADERS(net/pfvar.h, , , [#include <sys/types.h>
+#include <net/if_pflog.h>
 #include <sys/socket.h>
 #include <net/if.h>])
 if test "$ac_cv_header_net_pfvar_h" = yes; then
@@ -336,7 +337,7 @@
 	# We have BPF, so build valgrindtest with "make test".
 	#
 	VALGRINDTEST=valgrindtest
-elif test -r /dev/bpf0 ; then
+elif test -r /dev/bpf0 || test -r /usr/include/net/bpf.h ; then
 	V_PCAP=bpf
 
 	#
@@ -601,6 +602,7 @@
 #endif
 #include <net/bpf.h>
 		    ])
+	    LIBS="$LIBS -lrt"
 	;;
 
 hurd)
--- a/pcap-bpf.c
+++ b/pcap-bpf.c
@@ -29,6 +29,7 @@
 #endif
 #include <sys/socket.h>
 #include <time.h>
+#include <sys/kern/time.h>
 /*
  * <net/bpf.h> defines ioctls, but doesn't include <sys/ioccom.h>.
  *
